Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -50,7 +50,7 @@ the CRs to their prior state without even necessarily suffering cluster downtime
 
     ```console
     # Store the `CephCluster` CR settings. Also, save other Rook CRs that are in terminating state.
-    kubectl -n rook-ceph get cephcluster rook-ceph -o yaml > cluster.yaml
+    kubectl -n rook-ceph get cephcluster my-cluster -o yaml > cluster.yaml
```

Comment:
my-ceph-cluster

Reply:
I used the name that we have in cluster-test.yaml to keep it consistent